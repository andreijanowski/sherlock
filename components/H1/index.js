import styled from "styled-components";
import { Box } from "@rebass/grid";

const H1 = styled(Box).attrs({
  as: "h1",
  mb: 4,
  mt: 0
})`
  font-size: ${p => p.theme.fontSizes.f56};
  font-weight: 600;
  line-height: 1.15;
  color: rgb(${p => p.theme.colors.dark});
`;

export default H1;
