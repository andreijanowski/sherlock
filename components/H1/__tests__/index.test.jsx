/* eslint-env jest */
import React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import H1 from "..";

describe("H1 component", () => {
  const comp = shallow(<H1 />);

  it("renders correctly", () => {
    expect(comp).toMatchSnapshot();
  });
});
