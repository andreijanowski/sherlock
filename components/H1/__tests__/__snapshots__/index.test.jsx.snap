// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 component renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [
        Object {
          "as": "h1",
          "mb": 4,
          "mt": 0,
        },
      ],
      "componentStyle": ComponentStyle {
        "componentId": "H1-sros4y-0",
        "isStatic": false,
        "rules": Array [
          "box-sizing: border-box;",
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          "color:rgb(",
          [Function],
          ");font-weight:",
          [Function],
          ";font-size:",
          [Function],
          ";line-height:1.15;@media (min-width:",
          [Function],
          "){font-size:",
          [Function],
          ";}@media (min-width:",
          [Function],
          "){font-size:",
          [Function],
          ";}",
        ],
      },
      "displayName": "H1",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "propTypes": Object {
        "bg": [Function],
        "color": [Function],
        "fontSize": [Function],
        "m": [Function],
        "mb": [Function],
        "ml": [Function],
        "mr": [Function],
        "mt": [Function],
        "mx": [Function],
        "my": [Function],
        "p": [Function],
        "pb": [Function],
        "pl": [Function],
        "pr": [Function],
        "pt": [Function],
        "px": [Function],
        "py": [Function],
        "width": [Function],
      },
      "render": [Function],
      "styledComponentId": "H1-sros4y-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
